metric_learn/rca.py

```diff
@@ -105,25 +105,30 @@ def fit(self, X, chunks):
                     ' It has been deprecated in version 0.5.0 and will be'
                     ' removed in 0.6.0. Use "n_components" instead',
                     DeprecationWarning)
+
+    if self.pca_comps != 'deprecated':
+      warnings.warn('"pca_comps" parameter is not used.'
+                    ' It has been deprecated in version 0.5.0 and will be'
+                    'removed in 0.6.0', DeprecationWarning)
+
     X, chunks = self._prepare_inputs(X, chunks, ensure_min_samples=2)
 
-    # PCA projection to remove noise and redundant information.
-    if self.pca_comps is not None:
-      pca = decomposition.PCA(n_components=self.pca_comps)
-      X_t = pca.fit_transform(X)
-      M_pca = pca.components_
-    else:
-      X_t = X - X.mean(axis=0)
```

Comment:
why is this centering step gone?

Reply:
I guess because we should remove any pre-processing step, but I agree I didn't talk about it at all, maybe we should keep the `ChangedBehaviorWarning` message below, but rather replace "no longer trained on a preprocessed version" by "no longer trained on centered data by default", and encourage to use a "StandardScaler" if needed ?

Reply:
Fair enough (I double-checked and this centering is not part of standard RCA)
Maybe keep ChangedBehaviorWarning but change to "no longer center the data before training RCA" (no need to mention scaler I think)
And in the deprecation warning, add the fact that PCA preprocessing should now be done by the user

Finally, have you checked the influence of removing the centering step on the examples?